src/connectors/hotmixradio.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+Connector.playerSelector = '#new_ajax-player';
+
+Connector.artistSelector = '#hotmixPlayerSongArtist .ajax';
+
+Connector.getArtist = () => getScrollableText('#hotmixPlayerSongArtist');
+
+Connector.getTrack = () => getScrollableText('#hotmixPlayerSongTitle');
+
+//Connector.getAlbum = () => getScrollableText('#songalbum');
```

Comment:
Actually you only have to add a space here, after the `//` to resolve the travis error. This should be possible with Notepad++

Reply:
Thanks, I will just remove the line